src/services/objection.service.ts

```diff
@@ -31,6 +31,13 @@ export const createNewObjection = async (companyNumber: string, token: string, c
   return objectionId;
 };
 
+export const updateObjectionUserDetails = async (companyNumber: string, objectionId: string,
```

Comment:
All new lines of code we add should be tested with a unit test to ensure it functions properly. This new function will need a new unit test in objection.service.spec.unit.ts.

Reply:
Put a test in here to make sure the sdk is being called with the values provided.